types/tls_altnames.pp

```diff
@@ -0,0 +1,7 @@
+# a type to describe tls_altnames
+type K8s::Tls_altnames = Array[
+  Variant[
+    Stdlib::Fqdn,
+    Stdlib::IP::Address::Nosubnet,
```

Comment:
This feels like [Stdlib::Host](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/host.pp), but isn't quite that. Perhaps stdlib should start using the non-compat IP addresses instead.